kitsune/questions/views.py

```diff
@@ -583,6 +585,9 @@ def aaq(request, product_key=None, category_key=None, step=1):
 
         if form.is_valid() and not is_ratelimited(request, "aaq-day", "5/d"):
             if in_blocklist(form.cleaned_data["content"]):
+                if switch_is_active('spam_deactivate_user'):
```

Comment:
"blocklist_deactivates_users" might be a more specific term

Do we also need to set a default here, or some kind of migration? I'm not familiar with waffle

Reply:
If waffle doesn't find that switch returns False.